magenta/models/melody_rnn/melody_rnn_generate.py

```diff
@@ -223,9 +229,23 @@ def run_with_flags(generator):
   # files.
   date_and_time = time.strftime('%Y-%m-%d_%H%M%S')
   digits = len(str(FLAGS.num_outputs))
+
   for i in range(FLAGS.num_outputs):
     generated_sequence = generator.generate(input_sequence, generator_options)
 
+    if FLAGS.exclude_primer_midi:
+      # Remove the first half of the sequence. Since we are continually
```

Comment:
Depending on the `num_steps` flag and the length of the primer, removing the first half may not be correct. I would recommend removing notes from `generated_sequence` only if their `note.start_time` is earlier than `start_time`.